app/connectors/SubscriptionConnector.scala

```diff
@@ -22,10 +22,11 @@ import models.PsaSubscription.PsaSubscription
 import play.api.Logger
 import play.api.http.Status._
 import play.api.libs.json.{JsError, JsResultException, JsSuccess, JsValue}
-import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse}
-import uk.gov.hmrc.http.HttpClient
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpResponse, StringContextOps}
```

Comment:
not only this, a couple of more connectors are still referring old HttpClient in the imports
Please remove imports for HttpClient and URL (if not using)